libr/anal/p/anal_x86_cs.c

```diff
@@ -2403,6 +2409,10 @@ static void anop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len, csh
 			op->type = R_ANAL_OP_TYPE_PUSH;
 			op->cycles = CYCLE_REG + CYCLE_MEM;
 			break;
+		case X86_OP_REG:
+			op->type = R_ANAL_OP_TYPE_PUSH;
```

Comment:
this is TYPE_UPUSH, there's a bitmask for REG which is used for RJMP, and can be also used here, but will require adding the rpush type

Reply:
Added RPUSH which is UPUSH | REG. Should printed type be "upush" (like in rjmp, printed type is "ujump")?